scriptworker_client/tests/test_utils.py

```diff
@@ -171,8 +171,8 @@ def test_get_log_filehandle(path, tmpdir):
             True,
         ),
         (
-            ["bash", "-c", ">&2 echo bar && echo foo && exit -11"],
-            245,
+            ["bash", "-c", ">&2 echo bar && echo foo && kill -11 $$"],
```

Comment:
Clearly this test was wrong originally. https://github.com/mozilla-releng/scriptworker-scripts/pull/324 also cites some real world testing, which might be good to do if you haven't already.